src/extension/features/budget/link-to-inflows/index.js

```diff
@@ -0,0 +1,57 @@
+import { Feature } from 'toolkit/extension/features/feature';
+import { controllerLookup } from 'toolkit/extension/utils/ember';
+import { l10n } from 'toolkit/extension/utils/toolkit';
+import {
+  getSelectedMonth,
+  isCurrentRouteBudgetPage,
+  transitionTo,
+} from 'toolkit/extension/utils/ynab';
+
+/**
+ * Adds a click handler to the "TOTAL INFLOW" inspector area.
+ */
+export class LinkToInflows extends Feature {
+
+  injectCSS() {
+    return require('./index.css');
+  }
+
+  shouldInvoke() {
+    return isCurrentRouteBudgetPage();
+  }
+
+  onRouteChanged() {
+    if (this.shouldInvoke()) {
+      this.invoke();
+    }
+  }
+
+  /**
+   * Add the wrapper element to the TOTAL INFLOWS inspector area.
+   */
+  invoke() {
+    $('.budget-inspector')
+      .find(`h3:contains("${l10n('inspector.totalIncome', 'TOTAL INFLOWS')}")`)
```

Comment:
Actually I think this needs to be hardcoded to english as well... I'm testing in spanish right now and there's a localized string for `inspector.totalIncome`, but it's not what's used in the DOM, so this finds nothing. 

brb fix coming up